README.md

```diff
@@ -301,42 +387,65 @@ ownershipscheme = "my:" kind ["@" did]
 
-Re-delegating these to further DIDs in a chain MUST address the specific parent DID that owns that resource separated by an `@`. For instance: `my:*@did:key:z6MkiTBz1ymuepAQ4HEHYSF1H8quG5GLVVQR3djdX3mDooWp` selects all resources originating from the specified DID, and `my:mailto@did:key:z6MkiTBz1ymuepAQ4HEHYSF1H8quG5GLVVQR3djdX3mDooWp` selects email addresses from the DID. 
+``` abnf
+delegatescheme = "as:" did ":" kind
+kind = "*" / <scheme>
+```
 
-### 4.1.2 Action
+### 4.2.2 Action
 
-The action for `my:*` MUST be `*`.
+The action for `my:*` or `as:*` MUST be the [superuser action `*`](#41-superuser). Another ability would not be possible, since any other ability cannot be guarinteed to work across all resource types (e.g. it's not possible to `crud/UPDATE` an email address). Recall that the superuser action is special in that it selects the maximum possible action for any resource.
 
 ``` json
 {"with": "my:*", "can": "*"}
+{"with": "as:did:key:z6MkiTBz1ymuepAQ4HEHYSF1H8quG5GLVVQR3djdX3mDooWp:*", "can": "*"}
+
+{"with": "my:mailto", "can": "msg/SEND"}
+{"with": "as:did:key:z6MkiTBz1ymuepAQ4HEHYSF1H8quG5GLVVQR3djdX3mDooWp:mailto", "can": "msg/SEND"}
 ```
 
-For `my` capabilities scoped to some scheme, the action MUST be one normally associated with that resource. 
+For `my` and `as` capabilities limited to some scheme, the action MUST be one normally associated with that resource. As it belongs to every action heirarchy, this MAY be the [superuser action `*`](#41-superuser).
 
 ``` json
 {"with": "my:dns", "can": "crud/UPDATE"}
+{"with": "my:dns", "can": "*"}
 ```
 
-## 4.2 UCAN Addressing
+## Proof Field Redelegation
 
-### 4.2.1 `prf` Scheme
+### `prf` Scheme
 
 The `prf` URI scheme defines addressing for UCANs and their fields.
 
 ``` abnf
-prf = "prf" selector
+prf = "prf:" selector
 selector = "*" / 1*DIGIT
 ```
 
-`prf:*` represents all of the UCANs in the current proof scope. The witnesses in the `prf` field may be referenced by their index, starting at 0: `prf:0`
+`prf:*` represents all of the UCANs in the current proofs array. The witnesses for the current UCAN MAY be referenced by their index in the `"prf"` field. If selecting a particular witness (i.e. not the wildcard), then a zero-indexed MUST be used. The first UCAN would be selected by `prf:0`, the second by `prf:1`, and so on. By virtue of the indexing scheme, selections MUST be performed on the current UCAN only, and cannot recursively on nested witnesses.
 
-### 4.2.2 `prf` Actions
+Further selection of capabilities inside fo specific witnesses MUST NOT be a valid parsing of this URI. For instance, `prf:0:mailto` MUST NOT be a valid `prf` URI.
+
+### 4.3.2 `prf` Actions
+
+The `prf` scheme MUST accept the following action: `ucan/DELEGATE`. This redelegates all of the capabilities in the selected witness(es).
```

Comment:
The second sentence could be made stronger by repeating "action". Something like:

> This action redelegates all of the capabilities in the selected witness(es).